apps/pie-docs/src/components/checkbox-group/overview/events.json

```diff
@@ -0,0 +1,19 @@
+{
+  "headings": [
+    "Event",
+    "Type",
+    "Description"
+  ],
+  "rows": [
+    [
+      "`pie-checkbox-group-disabled`",
+      "`CustomEvent`",
+      "Triggered after the disabled state of the checkbox group changes."
```

Comment:
Can this event be heard outside of the checkbox group? I think the current implementation fires the event directly at/inside the children, which are listening for it.

Reply:
As discussed have removed now.